analyses/SC_cd_2010/01_prep_SC_cd_2010.R

```diff
@@ -0,0 +1,91 @@
+###############################################################################
+# Download and prepare data for `SC_cd_2010` analysis
+# © ALARM Project, June 2022
+###############################################################################
+
+suppressMessages({
+    library(dplyr)
+    library(readr)
+    library(sf)
+    library(redist)
+    library(geomander)
+    library(cli)
+    library(here)
+    devtools::load_all() # load utilities
+})
+
+# Download necessary files for analysis -----
+cli_process_start("Downloading files for {.pkg SC_cd_2010}")
+
+path_data <- download_redistricting_file("SC", "data-raw/SC")
```

Comment:
This should have an argument, `year = 2010`. Otherwise the default is `year = 2020` and it is currently downloading 2020 Census statistics I believe